### DISCUSSION TITLE
Could doctrine/lexer v3 be supported?

### DISCUSSION THREAD

**Discussion by pribeirojtm** [doctrine/doctrine2#10997] [Support Questions] [answered]
Currently I get lots of composer errors when installing doctrine/orm after installing symfony, because symfony pulls in doctrine/lexer v3 and doctrine/orm is not compatible and requires a downgrade.

`  Problem 3
    - doctrine/orm is locked to version 2.16.2 and an update of this package was not requested.
    - doctrine/orm 2.16.2 requires doctrine/lexer ^2 -> found doctrine/lexer[2.0.0, 2.1.0] but it conflicts with your root composer.json require (^3.0).
  Problem 4
    - doctrine/orm 2.16.2 requires doctrine/lexer ^2 -> found doctrine/lexer[2.0.0, 2.1.0] but it conflicts with your root composer.json require (^3.0).`
Thanks

**Comment by derrabus** [comment, marked as answer]
ORM 3 will support it. That being said, you can safely downgrade. You won't lose any functionality by doing that.
